src/mumble/Settings.h

```diff
@@ -207,7 +207,12 @@ struct Settings {
 	enum WindowLayout { LayoutClassic, LayoutStacked, LayoutHybrid, LayoutCustom };
 	enum AlwaysOnTopBehaviour { OnTopNever, OnTopAlways, OnTopInMinimal, OnTopInNormal };
 	enum ProxyType { NoProxy, HttpProxy, Socks5Proxy };
-	enum RecordingMode { RecordingMixdown, RecordingMultichannel };
+	enum RecordingMode {
+		RecordingMixdown,
+		RecordingMultichannel,
+		RecordingMultichannelAndTransport,
+		RecordingTransportStandalone
```

Comment:
Do these recording modes work when not using Jack?

Reply:
They *could*, if implemented by the respective `*Audio` class, otherwise `RecordingMultichannelAndTransport` produces the exact same output as `Multichannel` and `RecordingTransportStandalone` is a no-op.

Reply:
Alright - but for the time being they don't work with other backends. In that case, we should make sure that the UI only allows for these options if an audio backend is used that actually supports those modes. Maybe this requires adding query functions for this capabilities to the audio backend interface (or did you already add that? I already forgot xD)

Reply:
xD no worries, yeah I haven't added something of that sort in yet- but it should be easy peasy to do so. Will fix!